Crypto/Crypto/Presentation/Views/ProfileView.swift

```diff
@@ -24,13 +24,14 @@ struct ProfileView: View {
                 
                 Text("Orlando Nicolas Marchioli")
                     .padding(.top, 40)
-                    .font(.headline)
+                    .font(.title2)
+                    .bold()
                 Text("Developer")
                     .padding(.top, 40)
-                    .font(.headline)
+                    .font(.title2)
                 Text("CABA, Buenos Aires, Argentina")
                     .padding(.top, 40)
-                    .font(.headline)
+                    .font(.title2)
                 Spacer()
```

Comment:
Not needed but just a small tip, Enums also come in hand over here. You could use them to handle the different styles and then create a function to apply them to your texts. I've found a [small example](https://medium.com/@paereson/how-to-deal-with-custom-fonts-in-swift-and-swiftui-f7886a824ac8) if you wanna dig into this 😄